examples/dao/test/dao-app-flow.js

```diff
@@ -944,5 +948,30 @@ describe("DAO test", function () {
 		]) {
 			assert.isUndefined(proposalALsigAcc.getLocalState(appID, key));
 		}
+
+		//verify that the funds from proposalLsig can only be withdrawed back to the owner
+		const withdrawTx = [
+			{
+				type: types.TransactionType.TransferAsset,
+				sign: types.SignType.LogicSignature,
+				fromAccountAddr: proposalALsigAcc.address,
+				toAccountAddr: proposerA.address,
+				amount: 5,
+				lsig: proposalALsig,
+				assetID: govTokenID,
+				payFlags: {},
+			},
+		];
+		let proposerABalance = proposerA.getAssetHolding(govTokenID).amount;
+		let proposalLsigABalance = proposalALsigAcc.getAssetHolding(govTokenID).amount;
+
+		runtime.executeTx(withdrawTx);
```

Comment:
as discussed, w should have two transactions: 
- one withdrawing let's say 2 tokens to a new user (he must opt-in before)
- second: withdrawing  let's say 8 tokens to the owner.
Both tx should succeed and we need to add balance checks after each transaction.

Also, let's add a test with withdraw from not an owner - it should fail ( we can do it in this function, before the "positive" withdraws )